src/components/launcher/Launcher.tsx

```diff
@@ -125,17 +126,7 @@ const Launcher = () => {
 							return {
 								label: kernelSpecs.kernelspecs[key]!
 									.display_name,
-								icon: (
-									<img
-										src={`${kernelSpecs.kernelspecs[key]?.resources["logo-svg"]}`}
-										alt={
-											kernelSpecs.kernelspecs[key]!
-												.display_name
-										}
-										height="36px"
-										width="36px"
-									/>
-								),
```

Comment:
`kernelSpecs` is fetched from the user's environment so I would prefer to use that in cases where it is available. The reason is that a user might have another kernel other than a Python one installed. 

Maybe another framing of this could be:
```
icon: kernelSpecs.kernelspecs[key]
									?.resources ? (
									<img
										src={
											kernelSpecs.kernelspecs[key]
												?.resources["logo-svg"]
										}
										alt={
											kernelSpecs.kernelspecs[key]!
												.display_name
										}
										height="36px"
										width="36px"
									/>
								) : (
									<PythonIcon boxSize={"36px"} />
								)
```

In this way, it will fetch the SVG if it is available from the kernel and otherwise fetch the default Python icon which would be a reasonable fallback